pkg/machinery/resources/block/mount_status.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/cosi-project/runtime/pkg/resource/typed"
 
 	"github.com/siderolabs/talos/pkg/machinery/proto"
+	"github.com/siderolabs/talos/pkg/xfs"
```

Comment:
this is not possible to have - we can't have `machinery` import `talos`.

so we have two options here - make it `any`, or make it some interface, also see my other comment about adapters